neptune/new/handler.py

```diff
@@ -97,6 +174,33 @@ def log(self,
             timestamp: Optional[float] = None,
             wait: bool = False,
             **kwargs) -> None:
+        """Logs the provided value or a collection of values.
+
+        Available for following field types (`Field types docs page`_):
+            * `Integer`
+            * `Float`
+            * `String`
+            * `Collection`
+            * `File`
+
+
+        Args:
+            value:
+            step (float or int, optional, default is None): Index of the log entry being appended.
+                Must be strictly increasing.
+                Defaults to `None`.
+            timestamp(float or int, optional): Time index of the log entry being appended in form of Unix time.
+                If `None` current time (`time.time()`) will be used as a timestamp.
+                Defaults to `None`.
+            wait (bool, optional): If `True` the client will wait to send all tracked metadata to the server.
+                This makes the call synchronous.
+                Defaults to `False`.
+            **kwargs:  TODO: missing docs
```

Comment:
What's this?

Reply:
To jest jakiś parametr, który został dodany niedawno przez @aniezurawski  .
Na razie jest ignorowany, ale w przyszłości będzie używany przez ImageSeries.

Jaki opis tutaj dać?

Reply:
@shnela ok, to wiem co to jest. Wyciąć. Będzie w API Reference, ale nie musi być tu. To jest funkcjonalność pod wsteczną kompatybilność której nie chcemy eksponować.